### DISCUSSION TITLE
On Click Notification Action

### DISCUSSION THREAD

**Discussion by LoopFruits** [runelite/runelite#16781] [Ideas]
When a Runelite notification pops up, clicking it would take you to the client. 

For example, when I receive an idle notification or low prayer notification it could take me to the client rather than clicking the application in the taskbar.

**Comment by Nightfirecat** [comment]
To do this, we'd need to write our own custom popup notification system, as what we do currently is use your operating system's popup notifications; we're not especially interested in doing that, as we do not have any way to design/create them such that they would not overlap/overlay/interfere with native notifications. I recommend checking to see if your operating system has an option to focus the application which fired the clicked notification.
